src/components/dialogs/create-filter-dialog.js

```diff
@@ -238,62 +240,108 @@ const CreateFilterDialog = ({
     };
 
     return (
-        <Dialog
-            fullWidth={true}
-            open={open}
-            onClose={handleClose}
-            onKeyPress={handleKeyPressed}
-        >
-            <DialogTitle onClose={handleClose}>{title}</DialogTitle>
-            <DialogContent>
-                <TextField
-                    autoFocus
-                    margin="dense"
-                    type="text"
-                    style={{ width: '90%' }}
-                    onChange={(event) =>
-                        handleFilterNameChanges(event.target.value)
-                    }
-                    error={
-                        newNameList !== '' &&
-                        !filterNameValid &&
-                        !loadingCheckFilterName
-                    }
-                    label={inputLabelText}
-                />
-                {renderFilterNameStatus()}
-                <RadioGroup
-                    aria-label="type"
-                    name="filterType"
-                    value={newListType}
-                    onChange={(e) => setNewListType(e.target.value)}
-                    row
-                >
-                    <FormControlLabel
-                        value="FORM"
-                        control={<Radio />}
-                        label={<FormattedMessage id="FORM" />}
+        <>
+            <Dialog
+                fullWidth={true}
+                open={!isValidated && open}
```

Comment:
Maybe change open state when the dialog is validated so that isValidated is not used for dialog opening purpose